docs/develop/basics/hackathon-guide.md

```diff
@@ -4,23 +4,143 @@ title: Hackathon Startup Guide
+  are the most secure methods.
+
+#### 3. E-mail / Phone Number Account Recovery
+
+* Enter the account activation code that you received.
+
+#### 4. Success!
+
+* You just created a `testnet` account and received 200 Ⓝ! Upon recovery method confirmation
+  you should be directed to your account dashboard.
+
+
+### Installing the `near-cli`
+
+Now it's time to install [`near-cli`](/docs/tools/near-cli#setup). This is a command line interface
+that allows you to interact seamlessly with NEAR. [This page](/docs/tools/near-cli) has all of the
+`near-cli` commands with examples.
+
+The following instructions are taken from the `near-cli` [installation
+guide](https://docs.near.org/docs/tools/near-cli#setup). If you already have the command line
+interface, you can [skip these steps](#first-steps).
+
+> **Note:** Make sure you have a current version of `npm` and `NodeJS` installed.
+
+#### Linux and macOS
+
+1. Install `npm` and `node` using a package manager such as `nvm`. Sometimes there are issues
+   using Ledger due to how macOS handles node packages related to USB devices.
+   [[click here]](https://nodejs.org/en/download/package-manager/)
+2. Ensure you have installed Node version 12 or above.
+3. Install `near-cli` globally by running:
+
+```bash
+npm install -g near-cli
+```
+
+#### Windows
+
+> **Note:** For Windows users, we recommend using Windows Subsystem for Linux (`WSL`).
+
+1. Install `WSL` [[click here]](https://docs.microsoft.com/en-us/windows/wsl/install-manual#downloading-distros)
+2. Install ` Node.js` [[click here]](https://nodejs.org/en/download/package-manager/)
+3. Change `npm` default directory [[click here]](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally#manually-change-npms-default-directory)
+   - This is to avoid any permission issues with `WSL`
+4. Open `WSL` and install `near-cli` globally by running:
+
+```bash
+npm install -g near-cli
+```
+
+## First steps
+
+1) Now that you have an account, test out a simple `testnet` app and interact with the blockchain.
+Try [Counter](https://examples.near.org/rust-counter) or [Guest Book](https://examples.near.org/guest-book).
+
+2) Look around in [NEAR Explorer](https://explorer.testnet.near.org). Here you can search for all transactions
+and blocks produced on NEAR. Try searching for the account you just created and see the transactions you've
+created with Counter or Guest Book. 
+
+3) Try running your first `near-cli` command: [`near login`](/docs/tools/near-cli#near-login). This will
+redirect you to your NEAR Wallet and save your `testnet` account keys locally. _Look for them in a hidden file
+under your HOME folder (`~/.near-credentials`)_
+
+4) Try `create-near-app`! Run `npx create-near-app your-awesome-project` in your terminal. _(Note: this requires
+[Node.js](https://nodejs.org/en/).)_ This is the easiest way to launch a fullstack app on the NEAR blockchain
+in under 5 minutes.
```

Comment:
We should delete this line as we are moving away from `create-near-app`

Reply:
Removed 👍